### PR TITLE
Humans are now as good as human+

### PR DESCRIPTION
**Pull request by HugoLuman**
All racial balance issues are now solved simply by giving humans the same feature that the human+ races get:
image
Full color customization.

Signed-off-by: HugoLuman <qaggeler@gmail.com>

:cl:HugoLuman
tweak:humans now have the same options as human+ races
tweak:human fingernails now treated realistically
/:cl:

### CODE REVIEW

**Comment by LightFire53** [comment]
Inb4 Reddit

**Comment by tigercat2000** [comment]
I'm not sure I actually like this- full color customization on humans makes it much more difficult to get an appealing skin tone for a "normal" human.

**Comment by bigfatbananacyclops** [comment]
this is a meme PR.. right?

**Comment by HugoLuman** [comment]
I see I forgot a very important part of this balance fix. Hopefully that fixes it.

**Comment by taukausanake** [comment]
How about we just make body paint a thing and put it in the arrivals dorm area? It can be in the art vendor.
The tattoos are nice too; maybe we can get a tattoo karma job that's like the barber.
LEGO minifig race when?

As a guy who has nice tough nails that a girl would kill for I'm kind of against humans having claws. I don't know why

**Comment by SamHPurp** [comment]
cat girls when

**Comment by KasparoVy** [comment]
How you know the github is dead

**Comment by KingPhilipIII** [comment]
How you know medrp has died.

**Comment by outbools** [comment]
inb4 first to close this PR would be accused of racism on Reddit

File: code/modules/mob/living/carbon/human/species/station.dm [hunk 1]
```diff
@@ -8,9 +8,9 @@
 	language = "Sol Common"
 	species_traits = list(LIPS, CAN_BE_FAT)
 	clothing_flags = HAS_UNDERWEAR | HAS_UNDERSHIRT | HAS_SOCKS
-	bodyflags = HAS_SKIN_TONE | HAS_BODY_MARKINGS
+	bodyflags = HAS_SKIN_COLOR | HAS_BODY_MARKINGS
 	dietflags = DIET_OMNI
-	unarmed_type = /datum/unarmed_attack/punch
+	unarmed_type = /datum/unarmed_attack/claws	//Humans can grow their fingernails out IRL you know
```
Comment (outbools) [inline code/modules/mob/living/carbon/human/species/station.dm]: b-but that should be an option!

**Comment by KasparoVy** [comment]
@KingPhilipIII Beating the horse's skeleton at this point.

**Comment by SamHPurp** [comment]
@HugoLuman you might as well give them a load of night vision too

**Comment by Keluandrie** [comment]
Finally, I can live out my roleplay fantasy of being Majin Buu, and defeating everyone with saiyan hairstyles on the station.

**Comment by HugoLuman** [comment]
>Not playing Jeice. Or Zarbon

**Comment by tigercat2000** [comment]
The memes, they are too much!

**Comment by HugoLuman** [comment]
Whew, for a while there I thought people were taking this seriously.

**Comment by adamkad1** [comment]
that would be fun tho

**Pull request closed without merging**